tests/Http/Controllers/Api/Entries/Upload/Internal/UploadWebControllerBulkUploadsTest.php

```
@@ -0,0 +1,441 @@
+                    ->get()
+            );
+
+            $createdAtAfter = BranchEntry::where('uuid', $branchEntryPayloads[0]['data']['id'])->value('created_at');
+            $this->assertEquals(
+                $createdAtBefore,
+                $createdAtAfter->format('Y-m-d\TH:i:s.000\Z')
+            );
+        } catch (Throwable $e) {
+            $this->logTestError($e, $response);
+        }
+    }
```

PHPMD (2.15.0)

The method test_should_not_touch_created_at_on_bulk_upload_branch is not named in camelCase. (Controversial Rules)

(CamelCaseMethodName)